src/components/TripItem.jsx

```diff
@@ -19,20 +20,44 @@ import {
   getFormattedDuration,
   getModes,
   formatTripDistance,
-  getStartDate,
-  getMainMode
+  getStartDate
 } from 'src/lib/trips'
 import { computeCO2Trip } from 'src/lib/metrics'
-import { pickModeIcon, modeToColor } from 'src/components/helpers'
+import {
+  pickPurposeIcon,
+  purposeToColor,
+  pickModeIcon
+} from 'src/components/helpers'
 import TripDialogDesktop from 'src/components/Trip/TripDialogDesktop'
 
 const styles = {
   co2: { fontWeight: 700 }
 }
 
+const getIconStyle = (idx, icons) => ({
+  margin: idx !== 0 && idx < icons.length - 1 ? '0 4px' : '0'
+})
+
+const TripItemSecondary = ({ tripModeIcons, duration, distance }) => {
+  return (
+    <>
+      {tripModeIcons.map((tripModeIcon, idx) => (
+        <Icon
+          key={idx}
+          icon={tripModeIcon}
+          size={10}
+          style={getIconStyle(idx, tripModeIcons)}
+        />
+      ))}
+      {` · ${duration} · ${distance}`}
```

Comment:
i think I would have test the text printed according to trip mode.

Reply:
*Les tests sur les composants viendront* 👍  (globalement, les tests sur les composants vont demander un travail sur les mock plus poussé qu'aujourd'hui)